### ISSUE TITLE
Addition to Domain of Credit Value https://credreg.net/ctdl/terms#creditValue

### ISSUE THREAD

**Issue by jeannekitchens** [CredentialEngine/Schema-Development#842]
Add Credential and all Credential Subclasses to domain of Credit Value https://credreg.net/ctdl/terms#creditValue.

Currently, to use ValueProfile, a Condition Profile must be created.  This is a barrier.  State of Florida is going to publish a few thousand credentials.  They do not have data for a condition profile.  They do have data for ValueProfile.  They will not be able to publish credit values if they're forced to create condition profiles.

This will help numerous other organizations that simply need to indicate credit value and credit type with Credentials.

**Comment by philbarker** [comment]
I agree with Nate, it would be ambiguous to mix "requirements" and "values" properties, the first is what you need in order to earn something the other is what something is worth for earning another thing. If having creating the ConditionProfile object with just the creditValue property is a burden, then it would be better to add ValueProfile to the range of ceterms:requires -- but as Nate says this just pushes the burden from publishers to consumers.

**Comment by jeannekitchens** [comment]
I'm resurrecting this issue.  We know that consuming credit values via condition profile actually creates a burden to data consumers.  Fundamentally, the simple use case is I need to know how many credits a credential, learning program, course, learning opportunity earn by completing them.  Right now, consumers have to go through condition profile to get this basic, standard information when that information is applicable.  Just as we support being able to indicate there is not credit, we need to be able to indicate there is credit.  The solution is to add https://credreg.net/ctdl/terms#creditValue to Credential.  It's domains already include: [ceterms:AssessmentProfile](https://credreg.net/ctdl/terms/AssessmentProfile) [ceterms:ConditionProfile](https://credreg.net/ctdl/terms/ConditionProfile) [ceterms:Course](https://credreg.net/ctdl/terms/Course) [ceterms:CourseComponent](https://credreg.net/ctdl/terms/CourseComponent) [ceterms:LearningOpportunityProfile](https://credreg.net/ctdl/terms/LearningOpportunityProfile) [ceterms:LearningProgram](https://credreg.net/ctdl/terms/LearningProgram) [ceterms:TransferIntermediary](https://credreg.net/ctdl/terms/TransferIntermediary)

**Comment by siuc-nate** [comment]
>We know that consuming credit values via condition profile actually creates a burden to data consumers.

How so? I have not seen this.

>Fundamentally, the simple use case is I need to know how many credits a credential, learning program, course, learning opportunity earn by completing them.

Credentials don't "earn" credit. You earn credit by completing the requirements for a Credential.

>Right now, consumers have to go through condition profile to get this basic, standard information when that information is applicable.

Yes, just like all of the other requirements for a Credential.  It is not difficult to get to:
```
Credential.requires[0].creditValue[0].value
```

> Just as we support being able to indicate there is not credit, we need to be able to indicate there is credit.

I [objected to adding that to Credential when we did that](https://github.com/CredentialEngine/Schema-Development/issues/845#issuecomment-1252874356) for the same reasons I am opposed to this proposal now. Further blurring the lines between a Credential and its requirements does a disservice to the entirety of CTDL and makes it harder to use for both publishers and consumers.
